grafana-plugin/src/containers/RotationForm/ScheduleOverrideForm.tsx

```diff
@@ -280,4 +315,26 @@ export const ScheduleOverrideForm: FC<RotationFormProps> = (props) => {
       </VerticalGroup>
     </Modal>
   );
+
+  async function onResize() {
+    setOffsetTop(await calculateOffsetTop());
+
+    setDraggablePosition({ x: 0, y: 0 });
+  }
+
+  async function calculateOffsetTop() {
```

Comment:
I can see that the same function is declared in another component. Since it doesn't depend on anything fro within the component, can we move it to some common util file and avoid duplication?